core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2516,4 +2516,13 @@ package object config {
       .version("3.5.0")
       .booleanConf
       .createWithDefault(false)
+
+  private[spark] val UNREGISTER_OUTPUT_THRESHOLD_ON_FETCH_FAILURE =
+    ConfigBuilder("spark.files.fetchFailure.unRegisterOutputThreshold")
```

Comment:
If you want to use `spark.files.fetchFailure.` namespace, could you move this after the existing config?

https://github.com/apache/spark/blob/1e090a57f0c6aa342c032130bbf86f8a152b9fee/core/src/main/scala/org/apache/spark/internal/config/package.scala#L942-L949

Reply:
Done